src/skdh/context/motion_detect.py

```diff
@@ -1,27 +1,47 @@
+"""
+Motion detection for triaxial accelerometer data.
+
+Yiorgos Christakis
+Copyright (c) 2024, Pfizer Inc. All rights reserved.
+"""
+
+from warnings import warn
 from importlib import resources
 
 import numpy as np
-from scipy.signal import iirfilter, filtfilt
+from numpy import round
+from scipy.signal import iirfilter, filtfilt, butter, sosfiltfilt
 
 from skdh.base import BaseProcess, handle_process_returns
 from skdh.utility import moving_mean, moving_sd
+from skdh.utility.internal import apply_resample
 
 
 def _resolve_path(mod, file):
     return resources.files(mod) / file
 
 
-class MotionDetectionAlgorithm(BaseProcess):
+class MotionDetectMahadevanEtAl(BaseProcess):
     """
-    Detect periods of motion from an accelerometer signal. Threshold approach on computed rolling coefficient
+    Detect periods of motion from an accelerometer signal. Threshold approach on computed 1s rolling coefficient
     of variation (CoV).
 
+    Method implemented as described in:
```

Comment:
Use the References section - see GaitLumbar for example

Reply:
Done